clang/test/Driver/print-enabled-extensions/aarch64-a64fx.c

```diff
@@ -0,0 +1,24 @@
+// REQUIRES: aarch64-registered-target
```

Comment:
these should probably all go in an `aarch64` folder with a `lit.local.cfg`:

```
targets = set(config.root.targets_to_build.split())
if not "AArch64" in targets:
    config.unsupported = True
```

and then drop the `REQUIRES:` lines from them.

Reply:
I've kept them as-is so that they still correspond to the [generating script](https://gist.github.com/tmatheson-arm/333dd14cc1c95ab4ac563ed615add95d).